dbt_project.yml

```diff
@@ -1,15 +1,20 @@
 name: 'klaviyo'
-version: '0.9.0'
+version: '1.0.0'
 config-version: 2
 require-dbt-version: [">=1.3.0", "<2.0.0"]
+
+dispatch:
+  - macro_namespace: dbt_utils
+    search_order: ['spark_utils', 'dbt_utils']
```

Comment:
I put this here because of [this wild line](https://github.com/fivetran/dbt_klaviyo_source/blob/main/dbt_project.yml#L17) in the source dbt_project.yml, but could appreciate a look at this.